CONTRIBUTING.md

```diff
@@ -25,12 +25,6 @@ reproducible) and a screenshot (if applicable).
 [bug reporting guidelines]: https://www.google.com/search?q=reporting+bugs
 
 
-Human interface guidelines 
---------------------------
-
-Use title case for titles, push buttons, menu titles and menu options. Elsewhere, use sentence case.
```

Comment:
It's ok to link Crapple's HIG. It would be even better if the link was made, literally at will:
```md
... please conform to [Crapple's HIG]. ...

[Crapple's HIG]: https://developer.apple.com/library/mac/documentation/UserExperience/Conceptual/OSXHIGuidelines/TerminologyWording.html
```
In either case, leave the above innocuous sentence be.